src/main/resources/alfresco/repository.properties

```diff
@@ -1327,4 +1327,8 @@ system.delete_not_exists.batchsize=100000
 system.delete_not_exists.delete_batchsize=1000
 system.delete_not_exists.read_only=false
 system.delete_not_exists.timeout_seconds=-1
-system.prop_table_cleaner.algorithm=V2
\ No newline at end of file
+system.prop_table_cleaner.algorithm=V2
+
+# Configure the expiration time of the direct access url.
+# Note: It is up to the actual ContentStore implementation if it can fulfil this request or not.
+alfresco.content.directAccessUrl.expiryTime.seconds=-1
```

Comment:
What does `-1` mean? No expiry time?

Reply:
I don't think it should be possible to get a link that never expires (not even sure the AWS API allows for that) so we should have a reasonable default i.e. 24 hours?

Reply:
The initial intent was that -1 - would imply not set and allowed for each connecter default to be used. But I can change it to 24 hours.

Reply:
Amazon sdk has something similar and defaults to 15 minutes if not set...

Reply:
Set to 86400 seconds (24 hours)

Reply:
Updated to 28800 (8 hours) - after talking with Alex Lee